sandbox/Sandbox/Generated.cs

```diff
@@ -1441,7 +1410,7 @@ public void Serialize(ref global::MessagePack.MessagePackWriter writer, global::
                     case 18:
                         if (!global::System.MemoryExtensions.SequenceEqual(stringKey, GetSpan_AdditionalProperty().Slice(1))) { goto FAIL; }
 
-                        __AdditionalProperty__ = formatterResolver.GetFormatterWithVerify<global::System.Collections.Generic.IDictionary<string, string>>().Deserialize(ref reader, options);
+                        formatterResolver.GetFormatterWithVerify<global::System.Collections.Generic.IDictionary<string, string>>().Deserialize(ref reader, options);
```

Comment:
What happened here? We're deserializing a dictionary and dropping the result on the floor. 
Before this, it looks like we were assigning to a local that would then be ignored. But why? 
If we don't actually use this value, why not call `reader.Skip` to save ourselves the trouble of deserializing it?

Reply:
The original `AdditionalProperty` is not writable and `__AdditionalProperty__` is discarded.

> If we don't actually use this value, why not call reader.Skip to save ourselves the trouble of deserializing it?

You are right.
[reader.Skip() can cause a deserializing matter.](https://github.com/neuecc/MessagePack-CSharp/issues/57#issuecomment-701120669)
For the safety of deserialization, I left `formatterResolver.GetFormatterWithVerify<global::System.Collections.Generic.IDictionary<string, string>>().Deserialize(ref reader, options);` as it was.

I wonder whether [reader.Skip() can be safely used with the rewind deserialization](https://github.com/neuecc/MessagePack-CSharp/issues/57#issuecomment-711085660).

I agree that this behaviour is forward looking to supporting circular references.
I rewrite it.

Reply:
We're going to Skip things we don't recognize regardless (e.g. map or array entries for which we have no matching property). I don't think we should behave differently for this particular case of a non-writable property just in case it contains something that will be referenced later. That doesn't help us document behavior that it will "always" work. The story will either be "it works if we actually deserialized the original referenced item" or "it always works", the latter being if we know how to 'rewind'. So I think we should Skip() for now for speed.

Reply:
Okay, it is a good reason to `Skip()`. The rewrite is done.